src/widgets/test-results/trendline/trendline.component.tsx

```diff
@@ -0,0 +1,266 @@
+
+  console.log({ panelUuid, testUuid, sortedObs });
+
+  if (!loaded || error) {
+    return null;
+  }
+
+  const panel = Object.entries(sortedObs).find(
+    ([, { uuid }]) => uuid === panelUuid
+  );
+
+  switch (panel[1].type) {
+    case "LabSet":
+      const data = panel[1].entries
+        .flatMap(x => x.members.filter(y => y.conceptClass === testUuid))
+        .sort(
+          (ent1, ent2) =>
+            Date.parse(ent2.effectiveDateTime) -
+            Date.parse(ent1.effectiveDateTime)
+        );
+      return [data[0].name, data, panel[0]];
+    case "Test":
+      return [
+        panel[0],
+        panel[1].entries.sort(
+          (ent1, ent2) =>
+            Date.parse(ent2.effectiveDateTime) -
+            Date.parse(ent1.effectiveDateTime)
+        ),
+        undefined
+      ];
+  }
+};
+
+type TrendlineParams = {
+  patientUuid: string;
+  panelUuid: string;
+  testUuid: string;
+};
+
+const TrendLineBackground = ({ ...props }) => (
+  <div {...props} className={styles["Background"]} />
+);
+
+const withPatientData = WrappedComponent => ({
+  patientUuid,
+  panelUuid,
+  testUuid,
+  openTimeline: openTimelineExternal
+}) => {
+  const patientData = useTrendlineData({ patientUuid, panelUuid, testUuid });
+  const openTimeline = React.useCallback(
+    () => openTimelineExternal(panelUuid),
+    [panelUuid]
+  );
+
+  if (!patientData) return <div>Loading...</div>;
+
+  return (
+    <WrappedComponent patientData={patientData} openTimeline={openTimeline} />
+  );
+};
+
+const DateFormatOption: Intl.DateTimeFormatOptions = {
+  month: "short",
+  day: "numeric"
+};
+const TableDateFormatOption: Intl.DateTimeFormatOptions = {
+  year: "numeric",
+  month: "short",
+  day: "numeric"
+};
+const TableTimeFormatOption: Intl.DateTimeFormatOptions = {
+  hour: "2-digit",
+  minute: "2-digit"
+};
+
+const TrendlineHeader = ({ openTimeline, title }) => (
+  <div className={styles["header"]}>
+    <p onClick={openTimeline} className={styles["back-button"]}>
+      <ArrowLeft24></ArrowLeft24> Back to timeline
+    </p>
+    <div className={styles["title"]}>{title}</div>
+  </div>
+);
+
+const Trendline: React.FC<{
+  patientData: ReturnType<typeof useTrendlineData>;
+  openTimeline: () => void;
+}> = ({ patientData, openTimeline }) => {
+  const leftAxisLabel = patientData?.[1]?.[0]?.meta?.units ?? "";
+
+  const data = [];
+  const tableData = [];
+
+  let dataset = patientData[0];
+
+  patientData[1].forEach(entry => {
+    const range =
+      exist(entry?.meta?.hiNormal) && exist(entry?.meta?.lowNormal)
+        ? {
+            max: entry.meta.hiNormal,
+            min: entry.meta.lowNormal
+          }
+        : {};
+
+    data.push({
+      date: new Date(Date.parse(entry.effectiveDateTime)),
+      value: entry.value,
+      group: dataset,
+      id: entry.id,
+      ...range
+    });
+
+    tableData.push({
+      date: new Date(Date.parse(entry.effectiveDateTime)).toLocaleDateString(
```

Comment:
We can / should use the "default" date formatting options coming from `dayjs` / `esm-framework`.